<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="psiLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; font-style:italic;&quot;&gt;PSI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="volumeLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="volumelcdNumber">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
selection-background-color: rgb(83, 255, 35);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
  </widget>
  <widget class="QLabel" name="cmcubiclabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;cubicCm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="heightprogressBar">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>41</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>16</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="pressureLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>44</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pressure</string>
   </property>
  </widget>
  <widget class="QLabel" name="densityLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>39</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Density</string>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>41</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Full</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdpressureNumber">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRoundProgressBar" name="densityroundprogressBar" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>261</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRoundProgressBar</class>
   <extends>QWidget</extends>
   <header>QRoundProgressBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
